doc/technotes/localeModels.rst

```diff
@@ -96,8 +97,8 @@ Qthreads thread scheduling
 
 When qthreads tasking is used, different Qthreads thread schedulers are
 selected depending upon the ``CHPL_LOCALE_MODEL`` setting.  For the flat
-locale model the "nemesis" thread scheduler is used, and for the NUMA
-locale model the "distrib" thread scheduler is used.  This selection is
+locale model the "nemesis" thread scheduler is used, and for the NUMA and KNL
+locale models the "distrib" thread scheduler is used.  This selection is
 done at the time the Qthreads third-party package is built, and cannot
 be adjusted later, either at user compile time or at execution time.
 
```

Comment:
I've fixed chplenv.rst and am about to push that change.  I agree that localeModels.rst and knl.rst need additional work before the doc freeze.  I've created issue #5569 to track changes such as this that need to be made after the KNL locale model commit.